include/El/core/environment/decl.hpp

```diff
@@ -265,6 +265,24 @@ DEBUG_ONLY(
     typedef CallStackEntry CSE;
 )
 
+#ifndef EL_RELEASE
+    void LogFileOpen( char* filename );
```

Comment:
Also, I would suggest using `const char* filename` since the string does not need to be changed.